packages/contracts/contracts/optimistic-ethereum/OVM/accounts/OVM_ECDSAContractAccount.sol

```diff
@@ -100,7 +100,6 @@ contract OVM_ECDSAContractAccount is iOVM_ECDSAContractAccount {
             "Fee was not transferred to relayer."
         );
 
-        // Contract creations are signalled by sending a transaction to the zero address.
```

Comment:
This is the only code changed to the contracts Solidity source code from `master` so far. However, the `contracts` package was bumped from `0.3.0` to `0.3.2`.

Reply:
There was an update to the state dump scripts